tox.ini

```diff
@@ -0,0 +1,14 @@
+[testenv]
+deps = pipenv
+setenv =
+    PIPENV_VERBOSITY=-1
+commands =
+    pipenv install --dev
+    pipenv run pytest tests
```

Comment:
Since tox _already_ creates its own environments, will `pipenv` integrate with it properly here? I have not tried to combine them before...

Reply:
Yes it does, pipenv detects that it's running inside virtual environment.
Warnings about that are silenced by `PIPENV_VERBOSITY` envvar.

Reply:
good to know 👍